src/darwin/mod.rs

```diff
@@ -216,21 +198,6 @@ impl Rasterizer {
             },
         }
     }
-
-    /// Helper to try and get a glyph for a given font. Used for font fallback.
-    fn maybe_get_glyph(
-        &self,
-        glyph: GlyphKey,
-        font: &Font,
-    ) -> Option<Result<RasterizedGlyph, Error>> {
-        let scaled_size = self.device_pixel_ratio * glyph.size.as_f32_pts();
```

Comment:
Huh I was kinda curious where this line has gone... but looks like it was never used for anything in the first place?

Reply:
Yeah, it was even silenced before. just strange...